app/assets/javascripts/utilities/getCsrfToken.js.erb

```diff
@@ -10,12 +10,7 @@ function getCsrfToken() {
         return resolve(authToken);
       } else if (i === 1000) {
         clearInterval(waitingOnCSRF);
-        var airbrake = new airbrakeJs.Client({projectId: 1, projectKey: '<%=ApplicationConfig["AIRBRAKE_API_KEY"]%>'});
-        airbrake.addFilter(function(notice) {
-          notice.context.environment = "<%= Rails.env %>";
-          return notice;
-        });
-        airbrake.notify("Could no locate CSRF metatag" + JSON.stringify(localStorage.current_user));
+        Honeybadger.notify("Could not locate CSRF metatag" + JSON.stringify(localStorage.current_user));
```

Comment:
This probably needs a space at the end of the string to avoid joining two strings: `Could not locate CSRF meta tag `